Add user stories  id_reference carrier (#233)
PrestaShop/prestashop-specs: content/1.7/back-office/shipping/carriers/add-edit.md

## Behaviors / User stories

> When you edit a carrier a new carrier is created. a link is saved to the original id with the id_reference.

Reviewer: ```suggestion
When you edit a carrier, a new carrier is created and the carrier id changes. A link is saved to the original id with the id_reference.
```
Hello @okom3pom and thank you for this PR
Could you tell me more about the link saved: where is it saved?

## Behaviors / User stories

> When you edit a carrier a new carrier is created it keeps a link with the old one with the id_reference

Reviewer 2: ```suggestion
When you edit a carrier a new carrier is created. a link is saved to the original id with the id_reference.
```
